java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Add the possibility to disable notifications types through a configuration parameter: java.notifications_type_disabled (bsc#1111910)
```

Comment:
I think we have a *soft* line-length limit for changelog messages (IIUC it is marked by the end of dashes).

Let's summarize it with
`- Disable notification types with 'java.notifications_type_disabled' in rhn.conf (bsc#1111910)`